src/opentime/rationalTime.h

```diff
@@ -77,6 +77,34 @@ class RationalTime
         return fabs(value_rescaled_to(other._rate) - other._value) <= delta;
     }
 
+    // Return whether the value and rate are equal to another RationalTime.
+    // This is different from the operator "==" that rescales the time before
+    // comparison.
+    constexpr bool exactly_equal(RationalTime other) const noexcept
+    {
+        return _value == other._value && _rate == other._rate;
+    }
+
+    // Return a RationalTime with the largest integer value not greater than
+    // this value.
+    RationalTime floor() const
```

Comment:
I almost wonder, to make the method names more self-describing, if they might be named `floor_to_rate`, `ceil_to_rate`, and `round_to_rate`.

Reply:
The ```_to_rate``` sounds a bit confusing to me, maybe ```floor_value```, ```ceil_value```, and ```round_value```?

Reply:
I think `floor_value` is better, since it tells me which field is changing.

Reply:
I can go for that :)